charts/beyla/templates/configmap.yaml

```diff
@@ -27,9 +27,9 @@ data:
     {{- if eq .Values.preset "application" }}
     {{- if not .Values.config.data.discovery }}
     discovery:
-      services:
+      instrument:
         - k8s_namespace: .
```

Comment:
I think we need to use `*` instead of a dot, the new instrument section is Glob.